debezium-connector-mariadb/src/main/java/io/debezium/connector/mariadb/MariaDbConnectorTask.java

```diff
@@ -78,6 +80,7 @@ protected Iterable<Field> getAllConfigurationFields() {
 
     @Override
     protected ChangeEventSourceCoordinator<MariaDbPartition, MariaDbOffsetContext> start(Configuration configuration) {
+        final MariaDbCharsetRegistry charsetRegistry = new MariaDbCharsetRegistry();
```

Comment:
Is this still needed?

Reply:
Good catch, I must have missed that in the refactoring :+1: